pythainlp/corpus/core.py

```diff
@@ -88,6 +88,24 @@ def get_corpus(filename: str) -> frozenset:
     return frozenset(lines)
 
 
+def _update_db(name, path):
+    local_db = TinyDB(corpus_db_path())
+    query = Query()
+    local_db.update({"filename": path}, query.name == name)
+    local_db.close()
+
+def _update_all():
+    print("Update Corpus...")
+    local_db = TinyDB(corpus_db_path())
+    item_all = local_db.all()
+    local_db.close()
```

Comment:
I think it is better to use `with` to handle IO related to variable.   To illustrate, if L100 is failed, `local_db` might not release the TinyDB resource. This is because the `close` command isn't executed because the program is failed at L100.

In this case, we should use
```
with TinyDB(...) as local_db:
    # do something.
```

See detailed explanation: https://stackoverflow.com/a/2738468.